### ISSUE TITLE
Attributes JSON and hombot camera in HA

### ISSUE THREAD

**Issue by dansh1n** [plakna/ha-lg-hombot#3]
1. Hi, you plan to display attributes JSON (lastclean, speed, mode. repeat)?

Now I get these statuses using command_line:
```
- platform: command_line
  unique_id: 'lg_hombot_mode'
  command: curl GET http://192.168.0.5:6260/status.txt | awk 'NR==14' | awk -F'=' '{print $2}' | awk -F'"' '{print $2}'
  name: LG HomBot Mode
```
2. Is it possible to get an image from the robot's camera in HA?

**Comment by plakna** [comment]
Yes, I will add the attributes when I have time. I don't think it's possible to extract the image, but I will take a look at it.

**Comment by dansh1n** [comment]
Thanks, it will be nice if you add attributes. Will wait.

**Comment by plakna** [comment]
The first option is now available. Users can adjust the turbo mode (fan speed) :)

The vacuum model in Home Assistant doesn't support different cleaning modes such as zigzag or cell by cell mode. Maybe I'll just create sensors for these options.

Extracting the image seems possible. However, the format is quite strange. It is a "YUV" image. I need to figure out how to convert it into a compatible format for Home Assistant.

**Comment by dansh1n** [comment]
Very nice, great news. Thank you!

**Comment by aldweb** [comment]
Very nice integration for my old Hombot, thank you!
I second the kind request of getting access to the camera image, the cleaning modes, and even the cleaning results maps... if ever this was doable...
